wagtailio/static/sass/components/_comparison-table.scss

```diff
@@ -0,0 +1,42 @@
+.comparison-table {
+    align-items: start;
+
+    td:first-child {
+        font-weight: $weight--bold;
+    }
+
+    tbody {
+        grid-column: 2 / span 1;
+        grid-gap: 20px;
+
+        @include media-query(medium) {
+            grid-column: 2 / span 3;
+        }
+    
+        @include media-query(large) {
+            grid-column: 2 / span 7;
```

Comment:
Suggestion: update this to `grid-column: 2 / span 5` as we only have 5 columns for content at the large breakpoint